packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -311,6 +315,33 @@ export default function PostOptionsMenu({
     });
   }
 
+  const isReminderActive = useFeature(feature.readingReminder);
```

Comment:
We seem to have merged 2 different reminder experiments, the right one should be this one `bookmarkReminder`. 

Would u mind changing this and any other usages on `feature.readingReminder` @ilasw 🙏

Reply:
Also, we should use the `useConditionalFeature` here and use `isPostOptionsOpen` for the `shouldEvaluate` prop. This is to not enroll everyone under the experiment until they bookmark or have bookmarked a post in the past.

Reply:
Updated! ✅